api/envoy/service/ext_proc/v3alpha/external_processor.proto

```diff
@@ -74,29 +74,37 @@ message ProcessingRequest {
     option (validate.required) = true;
 
     // Information about the HTTP request headers, as well as peer info and additional
-    // properties. If "response_required" is set, the server must send back a
+    // properties. If "async_mode" is set, the server must send back a
```

Comment:
I just reread the `async_mode` field comments with a fresh pair of eyes and they are quite confusing. Can you try and clarify? I'm not sure here why `async_mode` basically replaces `response_required`.

Reply:
All I'm doing in this PR is fixing the comment, since we renamed the flag from "response_required" to "async_mode" a long time ago while reviewing the original protocol. However, I made a mistake in the original PR. I just language to say "Unless async mode..." which is clearer IMO.

I remember a discussion on the original PR and we agreed that "async_mode" was better than "response_required" but TBH I thought that on this particular message, "response_required" is easier for a developer to understand.

Reply:
I simplified the comments a bit so PTAL.